### ISSUE TITLE
Recruitable Legendary Pokemon

### ISSUE THREAD

**Issue by BurningBlaze** [PMDShift/Proposals#1]
## Summary
A method for recruiting legendary Pokemon that adapts to player story progression while maintaining challenge and legendary rarity.

## Description
This proposal describes a method for recruitable legendary Pokemon that is fair, challenging, and still maintains a degree of rarity for legendary Pokemon.

The core mechanic is the introduction of unique items that correspond to an individual legendary. This item (referred to as a "fragment" in this document, but can easily be renamed) is a stackable, non-droppable, non-tradable currency item. Each fragment type corresponds to a single legendary Pokemon. 

Example: a Sky Fragment can correspond to Rayquaza, while a Meteor Fragment can correspond to Deoxys. 

In addition, each legendary Pokemon will be assigned to one/more dungeons. While the legendaries themselves won't appear as wild Pokemon in those dungeons, the fragments that correspond to the legendary will appear. Thus, the player will be able to enter those dungeons and find additional fragments.

The ultimate goal is for the player to collect a configurable number of fragments. The number of total fragments can be adjusted for each legendary. Weaker legendaries can require fewer fragments, while stronger (and rarer) legendaries can require significantly more fragments.

Once the player collects the required number of fragments, they will need to reach the end of the dungeon corresponding to the legendary. Once there, rather than the standard boss battle starting, the legendary will appear. If the player has not yet recruited that legendary, the recruit rate will be set to 100%. If the player has already recruited the legendary, they will still be able to battle it, however, it will be unrecruitable.

Once the legendary is defeated, all collected fragments corresponding to the legendary will be removed. If the legendary was recruited, the player will be able to use it outside of the dungeon as though it were any other Pokemon.

The recruited legendary cannot be traded. It can, however, be released. If released, the player is able to repeat the process and recruit the legendary again. This method can be used for hunting for shiny variants of the legendary, as the spawned legendary will have the standard chance of being shiny. This ensures that certain legendaries (shiny Rayquaza) will be extremely rare, but possible to obtain.

**Comment by shayrl56** [comment]
What if i wanna hunt for shiny rayquaza, it ends up not being shiny and i dont wanna recruit it, but once i defeat it, all collected fragments corresponding to the legendary will be removed. So if i keep doing it again to hunt shiny, i have to collect all the fragments again ?

**Comment by BurningBlaze** [comment]
Yes, while difficult, it means shiny Rayquaza will be extremely rare (as it should be)

**Comment by shayrl56** [comment]
This gonna take tons of efforts to work on. But sounds fair enough

**Comment by WaterLoveI** [comment]
This idea seems fine as long as
1.Storage is increased for all those fragments,maybe should just be unlimited 😄 
2.For absolute end-game only, I suggest something like only people with "Guildmaster" explorer rank will be able to pick up the fragments

**Comment by shayrl56** [comment]
i think master *** is fine enough. Reaching 100k points for people who does not play much is quite a nightmare

**Comment by Zozanero7** [comment]
I like the idea, but imo there should be scenarios without the fragments but something more story / side story driven.  Like assembling the pieces to Exodia without all the pieces being the same item just to add some more depth. An in-between would be like an archaeologist giving you endless quests (with cooldowns) to gather materials to summon say one of the gen 3 golems. Basic, still grindy, less depth, but a lot more repeatable than entire story missions might be.

**Comment by Zozanero7** [comment]
> I like the idea, but imo there should be scenarios without the fragments but something more story / side story driven. Like assembling the pieces to Exodia without all the pieces being the same item just to add some more depth. An in-between would be like an archaeologist giving you endless quests (with cooldowns) to gather materials to summon say one of the gen 3 golems. Basic, still grindy, less depth, but a lot more repeatable than entire story missions might be.

In a response to WaterLovel's suggestion about guildmaster or a higher rank being able to pick things up, i personally dont like that mechanic.  I know some items already require a rank to pickup but imo it should be you need to have the proper rank to use/ equip it rather than have to leave it behind.

**Comment by Zozanero7** [comment]
> This idea seems fine as long as
> 1.Storage is increased for all those fragments,maybe should just be unlimited 😄
> 2.For absolute end-game only, I suggest something like only people with "Guildmaster" explorer rank will be able to pick up the fragments

I personally don't like that mechanic. I know some items already require a rank to pickup but imo it should be you need to have the proper rank to use/ equip the item rather than have to leave it behind. It just never feels good missing out on something rare

**Comment by BurningBlaze** [comment]
> 1.Storage is increased for all those fragments,maybe should just be unlimited

We're adding a new "Key Items" inventory, which will be used for items such as Drif-Flute (aka fly) and we can put fragments there. They won't take up any normal inventory room.

> 2.For absolute end-game only, I suggest something like only people with "Guildmaster" explorer rank will be able to pick up the fragments

That's a good idea, I'll add a note that we can specify custom requirements per fragment type (weaker legendary fragments need a lower rank). I don't think it would be appropriate to enforce for all legendaries, though.

**Comment by WaterLoveI** [comment]
> I personally don't like that mechanic. I know some items already require a rank to pickup but imo it should be you need to have the proper rank to use/ equip the item rather than have to leave it behind. It just never feels good missing out on something rare

If we're able to pick up the fragments at any rank then (unless the requirements are absurd) they'll be able to summon the legends almost immediately (maybe even more than once) after they've reach *insert whatever rank we decided on* as they'll have them stocked up (which is your point). What I want is some level of grind that is exclusive to high explorer rank to encourage them to keep playing because at Guildmaster or whatever they'll already have lv100s and rare items,I want them to have another goal to pursue so this is why I firmly believe it should be guildmaster only.

**Comment by SupaFresh** [comment]
I vote no. Recruitment of legendary pokemon could cause balancing issues. Also it feels kind of awkward to be able to recruit legendary pokemon. For me at least it breaks my immersion when this all powerful god joins my team to help me find perfect apples or something.
